apps/nextjs-website/next.config.js

```diff
@@ -11,6 +11,7 @@ const nextConfig = {
   transpilePackages: [
     '@pagopa/pagopa-editorial-components',
     '@pagopa/mui-italia',
+    '@mui/material'
```

Comment:
```suggestion
```

are you sure this is necessary?

Reply:
I had add `@mui/material` to transpilePackages due to this error: 
```
nextjs-website:dev: - error /home/merack/Downloads/developer-portal/node_modules/@mui/material/Stack/Stack.js:1
nextjs-website:dev: import _objectWithoutPropertiesLoose from "@babel/runtime/helpers/esm/objectWithoutPropertiesLoose";
nextjs-website:dev: ^^^^^^
nextjs-website:dev: 
nextjs-website:dev: SyntaxError: Cannot use import statement outside a module
nextjs-website:dev:     at internalCompileFunction (node:internal/vm:73:18)
nextjs-website:dev:     at wrapSafe (node:internal/modules/cjs/loader:1176:20)
nextjs-website:dev:     at Module._compile (node:internal/modules/cjs/loader:1218:27)
nextjs-website:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
nextjs-website:dev:     at Module.load (node:internal/modules/cjs/loader:1117:32)
nextjs-website:dev:     at Module._load (node:internal/modules/cjs/loader:958:12)
nextjs-website:dev:     at Module.require (node:internal/modules/cjs/loader:1141:19)
nextjs-website:dev:     at require (node:internal/modules/cjs/helpers:110:18)
nextjs-website:dev:     at @mui/material/Stack/Stack (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:2853:18)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at eval (webpack-internal:///../../node_modules/@pagopa/pagopa-editorial-components/dist/components/PreHeader/PreHeader.js:12:41)
nextjs-website:dev:     at ../../node_modules/@pagopa/pagopa-editorial-components/dist/components/PreHeader/PreHeader.js (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:2122:1)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at eval (webpack-internal:///../../node_modules/@pagopa/pagopa-editorial-components/dist/components/PreHeader/index.js:9:22)
nextjs-website:dev:     at ../../node_modules/@pagopa/pagopa-editorial-components/dist/components/PreHeader/index.js (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:2133:1)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at eval (webpack-internal:///../../node_modules/@pagopa/pagopa-editorial-components/dist/index.js:110:21)
nextjs-website:dev:     at ../../node_modules/@pagopa/pagopa-editorial-components/dist/index.js (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:2199:1)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at eval (webpack-internal:///./src/pages/[productSlug]/overview.tsx:10:93)
nextjs-website:dev:     at ./src/pages/[productSlug]/overview.tsx (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:362:1)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at eval (webpack-internal:///./node_modules/next/dist/build/webpack/loaders/next-route-loader/index.js?page=%2F%5BproductSlug%5D%2Foverview&absolutePagePath=.%2Fsrc%2Fpages%2F%5BproductSlug%5D%2Foverview.tsx&preferredRegion=&middlewareConfig=e30%3D!:19:93)
nextjs-website:dev:     at ./node_modules/next/dist/build/webpack/loaders/next-route-loader/index.js?page=%2F%5BproductSlug%5D%2Foverview&absolutePagePath=.%2Fsrc%2Fpages%2F%5BproductSlug%5D%2Foverview.tsx&preferredRegion=&middlewareConfig=e30%3D! (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:2309:1)
nextjs-website:dev:     at __webpack_require__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/webpack-runtime.js:33:42)
nextjs-website:dev:     at __webpack_exec__ (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:3116:39)
nextjs-website:dev:     at /home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:3117:28
nextjs-website:dev:     at Object.<anonymous> (/home/merack/Downloads/developer-portal/apps/nextjs-website/.next/server/pages/[productSlug]/overview.js:3120:3)
nextjs-website:dev:     at Module._compile (node:internal/modules/cjs/loader:1254:14)
nextjs-website:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
nextjs-website:dev:     at Module.load (node:internal/modules/cjs/loader:1117:32)
nextjs-website:dev:     at Module._load (node:internal/modules/cjs/loader:958:12)
nextjs-website:dev:     at Module.require (node:internal/modules/cjs/loader:1141:19)
nextjs-website:dev:     at require (node:internal/modules/cjs/helpers:110:18)
nextjs-website:dev:     at requirePage (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/require.js:156:12)
nextjs-website:dev:     at /home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/load-components.js:68:84
nextjs-website:dev:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
nextjs-website:dev:     at async loadComponentsImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/load-components.js:68:26)
nextjs-website:dev:     at async DevServer.findPageComponentsImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/next-server.js:762:36)
nextjs-website:dev:     at async DevServer.findPageComponents (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/dev/next-dev-server.js:1266:20)
nextjs-website:dev:     at async DevServer.renderPageComponent (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/base-server.js:1321:24)
nextjs-website:dev:     at async DevServer.renderToResponseImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/base-server.js:1365:32)
nextjs-website:dev:     at async DevServer.pipeImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/base-server.js:641:25)
nextjs-website:dev:     at async Object.fn (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/next-server.js:1139:21)
nextjs-website:dev:     at async Router.execute (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/router.js:315:32)
nextjs-website:dev:     at async DevServer.runImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/base-server.js:615:29)
nextjs-website:dev:     at async DevServer.run (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/dev/next-dev-server.js:897:20)
nextjs-website:dev:     at async DevServer.handleRequestImpl (/home/merack/Downloads/developer-portal/apps/nextjs-website/node_modules/next/dist/server/base-server.js:546:20) {
nextjs-website:dev:   digest: undefined
nextjs-website:dev: }

```
I think is caused when pass icon Name to features component instead of Mui Icon